internal/service/remotecfg/remotecfg.go

```diff
@@ -226,6 +227,9 @@ var _ service.Service = (*Service)(nil)
 func (s *Service) Run(ctx context.Context, host service.Host) error {
 	s.ctrl = host.NewController(ServiceName)
 
+	s.registerCollector()
+	defer s.unregisterCollector()
```

Comment:
I should probably know this by now, but does `Update(...)` end up with a call to `Run(...)`?

I'm wondering if this means that only the attributes that are present when the `remotecfg` service is started get sent in the `RegisterCollector` call and no further updates (say, for example, the initial config were updated)?

I'm not set on that being the desired behavior, but wondering if it's intended :smile: